packages/types/src/index.ts

```diff
@@ -179,6 +179,26 @@ export class EN10168Schema extends BaseCertificateSchema {
   Certificate: EN10168SchemaCertificate;
 }
 
+export interface EcocData {
+  Data: {
+    ObjectOfDeclaration: ObjectOfDeclaration[];
+  };
+}
+
+type ObjectOfDeclaration = {
+  Quantities: Quantities[];
+};
+
+type Quantities = {
+  Amount: number;
+  Unit: string;
+};
+
+export type Quantity = {
+  Unit: string;
+  Value: number;
+};
```

Comment:
These types only relates to E-COC so they scoped under a clear namespace, maybe simply by adding a prefix.
Also we should not reuse `Quantity` in extract-summary, the properties are pascalcased and we use camelcase in this lib.